modules/contribute/pages/create-branches.adoc

```diff
@@ -0,0 +1,82 @@
+= Create a Working Branch
+
+Always make a new working branch for your work, no matter how small the change.
+You can create a working branch in Atom or from the command line.
+
+[#base-branch]
+== Choose the Right Base Branch
+
+A base branch contains the original source documentation.
+You create working branches from a base branch and send pull requests to a base branch for integration.
+A working branch isolates changes that solve a single issue.
+A pull request submitted from a working branch that only contains closely related changes is easier to test, review, and merge.
+
+// LLP 12.13.21 CHANGE TO MAIN
+The Stargate documentation repositories maintain one base branch: master.
+
+At the current time, we only maintain a single version of the Stargate docs.
+
+[#work-branch-atom]
+== Create a Working Branch in Atom
+
+These instructions assume you've xref:install-git-and-editor.adoc#install-atom[installed Atom] and xref:set-up-repository.adoc[forked, cloned, and configured the upstream and origin remotes] of a documentation repository.
```

Comment:
Could we just use a prereq section with a list?

Reply:
fixed